### ISSUE TITLE
"folder with images" : syntaxe ?

### ISSUE THREAD

**Issue by 2Spartiate** [AliaksandrSiarohin/first-order-model#138]
Hi,

I managed to make it work with:
- Torch 1.0.0 as instructed 
- 256*256 jpg source image
- 256*256 mp4 driving vidéo

However, the result is not so great (almost no mouth movement, and overall onbvious face distorsion to move the eyes and eyebraws etc)

So i have converted my mp4 video into about 1000 png pictures (30+ secondes, 30+ fps = ajout 1000 frames), into a dedicated folder. 

However I must have overlooked something, because i can't find the exact syntaxe, un order to put a image folder as a parameter for "driving_video" location.

What should i use ?
I have tried:
 " /path/to/the/folder/ "
and
"/path/to/the/folder/* "

But nothing seem to work.

Thanks

Kind regards,

**Comment by AliaksandrSiarohin** [comment]
Are you talking about training time? At test time it mp4 can be used.

**Comment by 2Spartiate** [comment]
No, i mean :

In the Readme we can read the following:

"the videos can be in '.gif', '.mp4' or folder with images. We recommend the later, for each video make a separate folder with all the frames in '.png' format. This format is loss-less, and it has better i/o performance."

So i'm trying to provide a folder full of PNG images from a clip  in place of a mp4 clip.

**Comment by AliaksandrSiarohin** [comment]
This is a description of training dataset. Are you training?
